src/compat/posix/time/strptime.c

```diff
@@ -0,0 +1,327 @@
+/**
+ * @file strptime.c
+ * @brief writing linux function strptime for embox os
+ *
+ * author Tamara Deryugina <tamara.deryugina@gmail.com>
+ * @date 02.03.2020
+ */
+
+#define _XOPEN_SOURCE
+#include<stdio.h>
+#include<stdlib.h>
+#include<string.h>
+#include<time.h>
+#include<ctype.h>
+#include<math.h>
+
+int str_legth(const char *s) {
+	int count = 0;
+	while(s[count] != '\0') {
+		count++;
+	}
+	return count;
+ }
+
+int num_length(char *new_s) {
+	int count = 0;
+	while ((new_s[count] > 47) && (new_s[count] < 58)) {
```

Comment:
Please use literal consts instead of integer values to make code more readable

Reply:
I fixed it. Thank you!